api-server/src/common/models/user.js

```diff
@@ -162,6 +162,8 @@ export default function initializeUser(User) {
   User.definition.properties.rand.default = getRandomNumber;
   // increase user accessToken ttl to 900 days
   User.settings.ttl = 900 * 24 * 60 * 60 * 1000;
+  // Sets ttl to 900 days for mobile login created access tokens
+  User.settings.maxTTL = 900 * 24 * 60 * 60 * 1000;
```

Comment:
This has no effect. You should not depend on it, and remove this instead.

Reply:
It did seem to matter when I tested the new endpoint.  Without adding this I got back a shorter TTL.

Reply:
Also the cookie lifetime was shortened without this line

Reply:
(I unresolved this to make sure Mrugesh has a chance to see the discussion)